### PR TITLE
KUZ-591 Remove load-balancer functionalities from the proxy.

### PR DESCRIPTION
**Pull request by dbengsch**
:warning: Must be merged after #7

### CODE REVIEW

File: lib/core/KuzzleProxy.js [hunk 1]
```diff
@@ -38,22 +35,24 @@ function LoadBalancer () {
...
 };
 
 /**
  * Reads configuration files and initializes plugins
  */
-LoadBalancer.prototype.initPlugins = function () {
+KuzzleProxy.prototype.initPlugins = function () {
   var pluginsConfiguration = this.readPluginsConfiguration();
 
+  console.log(pluginsConfiguration);
```
Comment (ballinette) [inline lib/core/KuzzleProxy.js]: remaining debug trace to clean.

File: .proxyrc [hunk 2]
```diff
@@ -1,20 +1,20 @@
 {
+  "toto": "bingo",
```
Comment (ballinette) [inline .proxyrc]: WTF ?

File: lib/service/Backend.js [hunk 3]
```diff
@@ -81,6 +84,11 @@ Server.prototype.onMessage = function (data) {
       break;
     case 'httpPortInitialization':
       this.httpPort = message.data.httpPort;
+      console.error(`Backend HTTP port of ${this.socketIp} received : ${this.httpPort}`);
```
Comment (ballinette) [inline lib/service/Backend.js]: why log an error here ?

Comment (dbengsch) [inline lib/service/Backend.js]: you're absolutely right

File: .proxyrc (hunk 2 above)
Comment (dbengsch) [inline .proxyrc]: A nasty test about RC precedence that I left behind ... :(

File: lib/core/KuzzleProxy.js (hunk 1 above)
Comment (dbengsch) [inline lib/core/KuzzleProxy.js]: right

**Comment by ballinette** [comment]
+1

**Comment by stafyniaksacha** [comment]
+1

**Comment by scottinet** [comment]
+1

**Pull request merged**